mods/ts/rules/structures.yaml

```diff
@@ -169,6 +169,29 @@ GAPILE:
 	Power:
 		Amount: -20
 
+GAPAVE:
+	Tooltip:
+		Name: Pavement
+	Valued:
+		Cost: 75
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 30
+		Prerequisites: barracks
+	Building:
+		Adjacent: 4
+		TerrainTypes: Clear, DirtRoad, Rough
+		BuildSounds: place2.aud
+		AllowInvalidPlacement: true
+		Footprint: xx xx
+		Dimensions: 2,2
+	KillsSelf:
+		RemoveInstead: true
+	LaysTerrain:
+		Template: 671
```

Comment:
That is quite different.  Compare D2KResourceLayer (which does LAT tile selection for spice - like TS concrete) to the regular ResourceLayer (which uses multiple variants of a single full-cell tile - like D2K concrete).